app/Services/Api/ChapterImage/AddListChapterImageService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Services\Api\ChapterImage;
+
+use App\Interfaces\ChapterImage\ChapterImageRepositoryInterface;
+use App\Services\BaseService;
+use App\Traits\UploadFileImageTrait;
+use Exception;
+use Illuminate\Support\Facades\Log;
+
+class AddListChapterImageService extends BaseService
+{
+    use UploadFileImageTrait;
+    protected $chapterImageRepository;
+
+    public function __construct(ChapterImageRepositoryInterface $chapterImageRepository)
+    {
+        $this->chapterImageRepository = $chapterImageRepository;
+    }
+
+    public function handle()
+    {
+        try {
+            $imageNumber = 1;
+
+            foreach ($this->data['image'] as $image) {
+                $this->data['image_number'] = $imageNumber;
+                $this->data['url'] = $this->uploadFileImage($image, 'image_chapter');
+                $imageNumber++;
```

Comment:
tùm lum cách mà sao lại chọn cách này em ?

```suggestion
            foreach ($this->data['image'] as $index => $image) {
                $this->data['image_number'] = $index++;
                $this->data['url'] = $this->uploadFileImage($image, 'image_chapter');
```

```suggestion
            $this->data['image_number'] = 1;
            foreach ($this->data['image'] as $image) {
                $this->data['image_number']++;
                $this->data['url'] = $this->uploadFileImage($image, 'image_chapter');
```